src/coprocessor/dag/executor/mod.rs

```diff
@@ -130,11 +130,17 @@ impl Row {
     }
 }
 
-pub trait Executor {
+pub trait Executor: Send {
```

Comment:
Why it has to been `Send`? I think you can make it as a generic constrain?

Reply:
It's better to make it `Send` than writing `exec: Box<Executor + Send>` everywhere?

Reply:
You can use `type` to give it an alias.